melodic/distribution.yaml

```diff
@@ -7271,6 +7271,24 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/open_manipulator_msgs.git
       version: melodic-devel
     status: developed
+  open_manipulator_p_simulations:
+    doc:
+      type: git
+      url: https://github.com/ROBOTIS-GIT/open_manipulator_p_simulations.git
+      version: melodic-devel
+    release:
+      packages:
+      - open_manipulator_p_gazebo
+      - open_manipulator_p_simulations
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/ROBOTIS-GIT-release/open_manipulator_p_simulations-release.git
+      version: 1.0.0-1
+    source:
+      type: git
+      url: https://github.com/ROBOTIS-GIT-release/open_manipulator_p_simulations-release.git
```

Comment:
This URL is a problem; it should point to the source repository (`https://github.com/ROBOTIS-GIT/open_manipulator_p_simulations.git`).

Reply:
Thank you. I will send a new PR.